docs/install/tutorials/rpi.rst

```diff
@@ -251,11 +253,57 @@ You can use :guilabel:`CTRL` + :guilabel:`SHIFT` + :guilabel:`V` to paste text i
 Installing Kolibri
 ------------------
 
```

Comment:
@benjaoming Moved all the RPi installation instructions here, but I'm still unsure if we are keeping the ``cffi`` library or not...?

Reply:
I talked to @jredrejo about it, and I think it's a flaky/flimpsy build error that I've been testing with recently, where possibly the armv6 `cffi` library bundled with our C extensions got left out. But I cannot test it or verify it any further.

Reply:
So if you want a quick conclusion: Yes, remove it.

Reply:
Also because _it's supposed to work_ without this instruction anyways!